mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -36,13 +36,22 @@ type seedlistTest struct {
 }
 
 func TestInitialDNSSeedlistDiscoverySpec(t *testing.T) {
-	mtOpts := mtest.NewOptions().Topologies(mtest.ReplicaSet).CreateClient(false)
-	mt := mtest.New(t, mtOpts)
+	mt := mtest.New(t, noClientOpts)
 	defer mt.Close()
 
-	for _, file := range jsonFilesInDir(mt, seedlistDiscoveryTestsDir) {
+	mt.RunOpts("replica set", mtest.NewOptions().Topologies(mtest.ReplicaSet), func(mt *mtest.T) {
```

Comment:
Should these still have CreateClient(false)?

Reply:
Yeah, these don't need a Client to be constructed. Added.